src/main/java/com/gw2auth/oauth2/server/service/gw2/client/ChainedGw2ApiClient.java

```diff
@@ -69,6 +70,8 @@ public ResponseEntity<Resource> get(Duration timeout, String path, MultiValueMap
                     } else {
                         response = clientAndMetadata.client.get(Duration.between(now, timeoutAt), path, query, headers);
                     }
+                } catch (ResourceAccessException e) {
+                    // no-op
```

Comment:
I thought about this and it should effectively not be a responsibility of this class to catch handle this exception.

Its an implementation detail of `RestOperationsGw2ApiClient`, which previously would throw this exception in case of a clientside timeout (connect/read).

I updated the implementation of `RestOperationsGw2ApiClient` to handle this exception and return a `REQUEST_TIMEOUT` in this case and updated the implementation of this class to continue with the next client in case of a `REQUEST_TIMEOUT`